app/about_us/OurMission.js

```diff
@@ -0,0 +1,14 @@
+import styles from "./OurMission.module.css";
+
+const ourMissionData = {
+  description:
```

Comment:
For practicing this is great! Seems like you understand how to display variables in React.
And it is also completely okay to have this directly inside the HTML.